config/routes.rb

```diff
@@ -1,16 +1,20 @@
 Rails.application.routes.draw do
-  scope ':locale', locale: /#{I18n.available_locales.join('|')}/ do
-    # resources :projects, only: [:index, :show]
-    resources :people, path: 'team', only: [:index, :show]
-    resources :contact_forms, path: 'contact-forms', only: [:new, :create]
-    get :cieszyn, to: redirect('cieszyn/welcome')
+  defaults locale: :pl do
+    get :service, controller: 'cieszyn/welcome', action: :show
+
     namespace :cieszyn do
       resource :collaboration, only: :show, controller: :collaboration
       resource :rules, only: :show
       resource :warranty, only: :show, controller: :warranty
-      resource :welcome, only: :show, controller: :welcome
       resources :prices, only: :index
     end
+  end
+
+  scope ':locale', locale: /#{I18n.available_locales.join('|')}/ do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.